tests/base.py

```diff
@@ -0,0 +1,269 @@
+import unittest
+import os
+from tap_tester import menagerie, runner, connections, LOGGER
+from datetime import datetime as dt
+
+class MailchimpBaseTest(unittest.TestCase):
+    
+    start_date = ""
```

Comment:
I can see any usage of this except the start_date test and we can avoid the use of it in that test file as well.

Reply:
Changed the code as per standard